### PR TITLE
FE-1360: simplify tokens schema

### PR DESCRIPTION
**Pull request by tremarkley**
As more chains get added to the tokenlist our previous schema requirements become too much of a burden to maintain. This change only requires that at least one of the required chains is specified in the token schema.

Fixes #[FE-1360](https://linear.app/optimism/issue/FE-1360)

### CODE REVIEW

**Comment by tremarkley** [comment]
Current dependencies on/for this PR:
* master
  * **PR #299** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/299" target="_blank">Graphite</a> 
    * **PR #300** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/300" target="_blank">Graphite</a>  👈
      * **PR #301** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/301" target="_blank">Graphite</a> 
        * **PR #302** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/302" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/300?utm_source=stack-comment).

File: src/schemas.ts [hunk 1]
```diff
@@ -56,113 +56,26 @@ export const TOKEN_DATA_SCHEMA = {
...
+      properties: {
+        ethereum: TOKEN_SCHEMA,
+        optimism: TOKEN_SCHEMA,
+        kovan: TOKEN_SCHEMA,
+        'optimism-kovan': TOKEN_SCHEMA,
+        'base-goerli': TOKEN_SCHEMA,
+        goerli: TOKEN_SCHEMA,
+        'optimism-goerli': TOKEN_SCHEMA,
+      },
+      additionalProperties: false,
```
Comment (roninjin10) [inline src/schemas.ts:69]:
```suggestion
      additionalProperties: true,
```
I don't remember what this does but should it be allowing additional properties so it's looser in general?

**Review by roninjin10 — APPROVED** [review]

File: src/schemas.ts (hunk 1 above)
Comment (tremarkley) [reply src/schemas.ts:69]: That field means that if a property is listed that is not included in our schema, then validation fails. I think we probably want to keep this set to false. Reason being is that the generate script expects a certain set of networks and if we left this open ended, then we would need to modify that: https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/src/generate.ts#L38

**Comment by tremarkley** [comment]
@tremarkley queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/300).

**Comment by tremarkley** [comment]
@tremarkley merged this pull request with [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/300).

**Pull request merged**